vmngclient/api/partition_manager_api.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from typing import List
+
+from tenacity import retry, retry_if_result, stop_after_attempt, wait_fixed
+
+from vmngclient.api.repository_api import RepositoryAPI
+from vmngclient.utils.creation_tools import get_logger_name
+from vmngclient.utils.operation_status import OperationStatus
+
+logger = logging.getLogger(get_logger_name(__name__))
+
+
+class PartitionManagerAPI:
+    """
+    API methods for partitions actions. All methods
+    are exececutable on all device categories.
+    """
```

Comment:
https://google.github.io/styleguide/pyguide.html#384-classes

Reply:
Is it necessary? The difference is not significant, and IMO it looks more esthetically.